Stripe/PublicHeaders/UI/STPPaymentCardTextField.h

```diff
@@ -220,6 +220,11 @@
  */
 @property(nonatomic, readonly, nullable) STPCardParams *card;
```

Comment:
This should probably just update the property to be readwrite, rather than defining a separate method.